connect-and-run.md

```diff
@@ -0,0 +1,213 @@
+
+### Required setup for connection to SAS 9.4
+
+In order to configure the connection between VS Code and SAS 9, you must configure OpenSSH. Follow the steps below to complete the setup.
+#### Windows
+1. Enable openssh client optional feature; [instructions found here](https://learn.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse?tabs=gui).
+
+2. [Create an environment variable](https://phoenixnap.com/kb/windows-set-environment-variable) SSH_AUTH_SOCK with value //./pipe/openssh-ssh-agent (windows uses a named pipe for the auth sock).  
+**Note**: An attempt to create the varible using Powershell command line did not register; suggest using these GUI instructions.
+
+3. Ensure ssh-agent service is running and set startup type to automatic; commands found in [this link](https://dev.to/aka_anoop/how-to-enable-openssh-agent-to-access-your-github-repositories-on-windows-powershell-1ab8)  
+
+4. [Generate ed25519 keys](https://medium.com/risan/upgrade-your-ssh-key-to-ed25519-c6e8d60d3c54) with the following command (email address is not binding; use any):  
+
+```
+     ssh-keygen -o -a 100 -t ed25519 -f ~/.ssh/id_ed25519 -C "youremail@company.com"
+```
+5. You’ll be asked a series of questions. First, if you didn not provide a path, a default one is provided. Also, if you wish to add a passphrase enter it. Pressing the ‘Enter’ key for each question accepts the default key name and does not password protect your key.
+
+   - Enter a file in which to save the key (/c/Users/you/.ssh/id_ed25519):[Press enter]
+   - Enter passphrase (empty for no passphrase): [Type a passphrase]
+   - Enter same passphrase again: [Type passphrase again]  
+
+   
+6. Define an entry in ~/.ssh/config of the form:
+```
+    Host host.machine.name 
+        AddKeysToAgent yes
+        IdentityFile /path/to/private/key/with/passphrase 
+```
+Note: if ~/.ssh/config does not exist, run the following Powershell command to create it: `Out-File -FilePath config`  
+7. Add the private key to ssh-agent: ssh-add /path/to/private/key/with/passphrase
+8. In VS Code, define a connection profile  (see detailed instructions below in the [Add New SAS 9.4 Remote Profile](#add-new-sas-94-remote-profile) section). The connection for the remote server is stored in the settings.json file.
+```
+    "ssh_test": {
+        "connectionType": "ssh",  
+        "host": "host.machine.name",  
+        "saspath": "/path/to/sas/executable",  
+        "username": "username",  
+        "port": 22
+    }
+```
+Note: the default path to the SAS executable (saspath) is /opt/sasinside/SASHome/SASFoundation/9.4/bin/sas_u8. Check with your SAS administrator for the exact path.  
+7. Add the public part of the keypair to the SAS server. Add the contents of the key file to the ~/.ssh/authorized_keys file.
```

Comment:
looks like should be 9.

Reply:
Fixed items 7,8, and 9 so they now appear correctly, in order, on new lines.